### PR TITLE
Apply consistent naming for endpoints

### PR DESCRIPTION
**Pull request by ncjones**
Ticket Link: https://www.notion.so/immersve/Remove-a-from-endpoint-titles-83c254c7704e4d80a4a6c68772bd3887?pvs=4

### CODE REVIEW

File: imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ post:
   tags:
     - authentication
   summary: Auth Token Exchange
```
Comment (catluin) [inline imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml:4]: should it be called `Exchange Auth Token`? we're putting verbs first in all other endpoints names and this one stands out

File: imsv-docs-docusaurus/openapi/endpoints/currency/currency-convert.yaml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 get:
   tags:
     - currency
-  summary: Currency conversion
+  summary: Currency Conversion
```
Comment (catluin) [inline imsv-docs-docusaurus/openapi/endpoints/currency/currency-convert.yaml:4]: `Convert Currency`? not sure if this one is an exception or not

File: imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml (hunk 1 above)
Comment (ncjones) [reply imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml]: yes

File: imsv-docs-docusaurus/openapi/endpoints/currency/currency-convert.yaml (hunk 2 above)
Comment (ncjones) [reply imsv-docs-docusaurus/openapi/endpoints/currency/currency-convert.yaml:4]: agreed, renamed to "Get Currency Conversion"

**Review by catluin — APPROVED** [review]

**Pull request merged**